crazyswarm2/launch/launch_teleop2.py

```diff
@@ -0,0 +1,127 @@
+
+    # construct motion_capture_configuration
+    motion_capture_yaml = os.path.join(
+        get_package_share_directory('crazyswarm2'),
+        'config',
+        'motion_capture.yaml')
+
+    with open(motion_capture_yaml, 'r') as ymlfile:
+        motion_capture = yaml.safe_load(ymlfile)
+
+    motion_capture_params = motion_capture["/motion_capture_tracking"]["ros__parameters"]
+    motion_capture_params["rigid_bodies"] = dict()
+    for key, value in crazyflies.items():
+        type = crazyflie_types[value["type"]]
+        
+        motion_capture_params["rigid_bodies"][key] =  {
+                "initial_position": value["initial_position"],
+                "marker": type["marker"],
+                "dynamics": type["dynamics"],
+            }
+
+    # construct crazyswarm2_server configuration
+    server_yaml = os.path.join(
+        get_package_share_directory('crazyswarm2'),
+        'config',
+        'crazyswarm2_server.yaml')
+    
+    with open(server_yaml, 'r') as ymlfile:
+        server_params = yaml.safe_load(ymlfile)
+
+    server_params = server_params["/crazyswarm2_server"]["ros__parameters"]
+    server_params["crazyflies"] = crazyflies
+    server_params["crazyflie_types"] = crazyflie_types
+
+    # teleop params
+    teleop_yaml = os.path.join(
+        get_package_share_directory('crazyswarm2'),
+        'config',
+        'teleop.yaml')
+    teleop_5_yaml = os.path.join(
+        get_package_share_directory('crazyswarm2'),
+        'config',
+        'teleop_5.yaml')
+    
+    return LaunchDescription([
+        Node(
+            package='motion_capture_tracking',
+            executable='motion_capture_tracking_node',
+            name='motion_capture_tracking',
+            output='screen',
+            parameters=[motion_capture_params]
+        ),
+        # Node(
+        #     package='crazyswarm2',
+        #     executable='teleop',
+        #     name='teleop',
+        #     remappings=[
+        #         ('takeoff', 'cf231/takeoff'),
+        #         ('land', 'cf231/land'),
+        #         ('cmd_vel', 'cf231/cmd_vel'),
+        #         ('cmd_full_state', 'cf231/cmd_full_state'),
+        #         ('joy', 'cf231/joy'),
+        #     ],
+        #     parameters=[teleop_yaml]
+        # ),
+        Node(
+            package='crazyswarm2',
+            executable='teleop',
+            name='teleop',
+            remappings=[
+                ('takeoff', 'cf5/takeoff'),
+                ('land', 'cf5/land'),
+                ('cmd_vel', 'cf5/cmd_vel'),
+                ('cmd_full_state', 'cf5/cmd_full_state'),
+                ('joy', 'cf5/joy'),
+            ],
+            parameters=[teleop_5_yaml]
+        ),
+        # Node(
+        #     package='joy',
+        #     executable='joy_node',
+        #     name='joy_node',
+        #     remappings=[('joy', 'cf231/joy')],
+        #     output='screen',
+        #     parameters=[{'device_id':0}] 
+        # ),
```

Comment:
not supposed to be commented out?

Reply:
I created this one to be used for 2 robot cases for now, and left launch.py as it was before (just one robot case).